packer/bsd/bsd.pkr.hcl

```diff
@@ -97,11 +49,11 @@ source "qemu" "qemu-gce-builder" {
 }
 
 build {
-  name="netbsd-vanilla"
+  name = "vanilla"
   sources = ["source.qemu.qemu-gce-builder"]
 
   provisioner "shell" {
-    script = "scripts/bsd/netbsd-prep-gce.sh"
+    script = "${var.gce_script}"
```

Comment:
I think it might look nicer to address these with `only = ["whatever-netbsd-task's-name-is"]`?

Reply:
Isn't setting name for build and calling that build with -only enough?

Reply:
What I mean is that instead of using a variable for the script, you can do something like

```hcl
  provisioner "shell" {
    script = "scripts/bsd/netbsd-prep-gce.sh"
    only = ['netbsd'] #actually a different name here
  }
  provisioner "shell" {
    script = "scripts/bsd/openbsd-prep-gce.sh"
    only = ['openbsd'] #actually a different name here
  }
```
This isn't quite right, but should get the idea across.

Reply:
Done.